HHS/Head-Start-TTADP
# Requested changes to the TTA Overview widgets - add non-grantees, combine tta hours, fix region order

## Description of change
Follow-up PR to the TTA overview widget. Fixes region order in the region select dropdown. Adds non-grantees to the widget and combines duration into one item. Additionally, it includes the legacy reports in statistics and limits the reports to approved ones. This PR also creates a script that allows to extract the non-grantee data from the legacy imported field.


## How to test
- go to dev 
https://tta-smarthub-dev.app.cloud.gov/activity-reports
- assign yourself permissions to a lower number region (you might have to remove one first if having permissions to all)
- verify that the available regions in the region selectors are displayed in an ascending order
- add an activity report for a non-grantee
- verify that the widget was updated

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [n/a] Staging smoke test completed

### After merge/deploy
